defaults/clade_emergence_dates.tsv

```diff
@@ -43,3 +43,5 @@ Nextstrain_clade	first_sequence
 24A	2023-07-01
 24B	2023-11-01
 24C	2023-12-01
+24D	2024-02-01
```

Comment:
```suggestion
24D	2023-11-01
```

We should not make these dates too late because we use this date here as a cutoff for outliers - if something is sequenced before the emergence date we treat it as a date error.

I checked on covSpectrum and earliest are from early November - I'll assume these are real for now